EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+from typing import Callable, Optional
+import asyncio
+from ..exceptions import TaskFailedError
+
+_EPOCH = datetime.datetime(1970, 1, 1)
+__all__ = (
+    "TimeParser",
+    "Tasks",
+)
+
+
+
+
+@staticmethod
+def total_seconds(
+    day: int = 0,
+    hour: int = 0,
+    minute: int = 0,
+    seconds: float = 0,
+) -> float:
+    hour += day * 24
+    minute += hour * 60
+    seconds += minute * 60
+    return seconds
+
+
+class Tasks:
+    def __init__(self, client) -> None:
+        self.client = client
+
+    def add_task(
+        self,
+        task: Callable[..., None],
+        interval: Optional[int] = None,
+        **kwargs,
+    ):
+        """Adds a background task (Tasks that run silently in the background)
+
+        This task can be used to monitor the bot's performance, change and send stuff routinely
+        through webhooks, etc
+        You can set the time of when it starts, stops and  add which arguments to give to the task
+        You may also define whether to run once or run indefinitely
+
+        Args:
+            task (Callable): 
+            The function to run on the background
+            interval (Optional[int]): 
+            The interval (in seconds) for a delay between when the instance of the task stops and starts again.Defaults to 5.
+            You may use `TimeParser.total_seconds` method
+            instances (Optional[int]): 
+            The number of instances to start after delay. Usually unlimited
```

Comment:
Ah yes

Reply:
> Misleading description. How are you going to say the number of instances to start, usually unlimited? People will think the task will run like
> 
> ```python
> while True:
>     print("Task")
> ```

It is like that but with delays, i do not understand what is "misleading" here

Reply:
What he means by "misleading" is that people could think a task is run x time for every intervals

Reply:
> What he means by "misleading" is that people could think a task is run x time for every intervals

I quite did not understand this
\

Reply:
People could understand
```py
while True:
     for i in range(instances):
         task()

    sleep(...)
```

Instead of
```py
while instance:
    task()
    sleep(...)
    instance -= 1

```

Reply:
I got it . Will make a fix now